godot-core/src/log.rs

```diff
@@ -21,6 +22,7 @@ macro_rules! godot_warn {
     };
 }
 
+/// Throws a godot error
```

Comment:
This doesn't throw an error (what does that even mean, there are no exceptions in Rust or GDScript?).
Instead, it _prints_ one. Here, it would be interesting to see how it differentiates from `godot_print` and `godot_warn`.

The [upstream Godot documentation](https://docs.godotengine.org/en/stable/classes/class_@globalscope.html) can also be a good source of information, although in this particular case, the method doesn't exist as is.